WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -970,7 +963,7 @@ protected Address doInBackground(Double... args) {
         }
 
         protected void onPostExecute(@Nullable Address address) {
-            if (!isAdded() || address == null || address.getMaxAddressLineIndex() == 0) {
+            if (address == null || address.getMaxAddressLineIndex() == 0) {
```

Comment:
@nbradbury I added `isAdded` check before `getResources` usages in ae4b483b7c7e258a5b2c250ccf3030f0b236e79a but there doesn't seem to be one in this particular method. Am I missing something there?